CHANGELOG.rst

```diff
@@ -0,0 +1,18 @@
+===========
+ Changelog
+===========
+
+1.1 (2019-09-17)
```

Comment:
why not semver?   And relatedly, seems optimistic to be past 1.0 already!

Reply:
This is semver? Or at least as close to semver as I generally get.

Reply:
And I personally feel that the first version of any software should always be 1.0. SymPy spent years as 0.*.*, even though it was perfectly capable. I think when you start at 0.0.1, it creates this stigma to bumping to 1.0, because the software isn't "ready" yet. SciPy had the same problem https://docs.scipy.org/doc/scipy/reference/release.1.0.0.html#why-1-0-now. So whenever I create a new package, I always start at 1.0 as the first usable version. That tells people that the package is ready to use now, and it removes the pressure from me and other contributors from viewing it in a perpetual state of never being finished.

Reply:
Probably not the place to debate all this but semvar is more about what you promise to users.  Before `1.0.0`, you promise nothing.  After `1.0.0`.
  * bump major: incompatible API changes
  * bump minor: add functionality (backwards-compatible)
  * bump patch: bug fixes (backwards-compatible)
Taken from https://devhints.io/semver.

Note in particular, semvar encourages bumping major versions which otherwise people seem reluctant to do.

I like the idea of quickly glancing at updates for some stack and being able to quickly decide which ones I might need to worry about.

Anyway, sorry for off-topic!

Reply:
TL;DR add one more digit and I'll shut up ;-)

Reply:
Well I also don't really like semver. Maybe I'll write a blog post about it at some point.